package.json

```diff
@@ -38,7 +40,15 @@
     "build": "rm -rf output && rollup -c",
     "build:watch": "rollup -cw",
     "test": "vtex-test-tools test",
-    "test:watch": "vtex-test-tools test --watch"
+    "test:watch": "vtex-test-tools test --watch",
+    "lint": "tsc --noEmit && eslint --quiet --fix --ext ts,tsx src/",
+    "prepublishOnly": "run-s lint test build"
+  },
+  "husky": {
+    "hooks": {
+      "pre-commit": "run-s lint",
+      "pre-push": "run-s lint"
```

Comment:
Kinda redundant to have the lint at the `pre-push` hook, since you're guaranteeing files to be linted at the `pre-commit` hook 🤓

Reply:
I coppied the vtex model xD 🤓

Reply:
Should I leave the pre-push hook without linting?